app/scripts/modules/core/authentication/authentication.module.js

```diff
@@ -6,14 +6,20 @@ module.exports = angular.module('spinnaker.authentication', [
   require('./authentication.service.js'),
   require('../config/settings.js'),
   require('./authentication.initializer.service.js'),
-  require('./authentication.interceptor.service.js'),
-  require('./userMenu/userMenu.module.js'),
```

Comment:
I don't think you want to remove these dependencies - they're not being brought in anywhere else.